.travis.yml

```diff
@@ -50,10 +50,8 @@ install:
     - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew config; fi
     - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew list --versions; fi
     - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew update; fi
-    - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew install swig python3 icu4c jq; fi
+    - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew install jq swig python3 icu4c openssl; fi
```

Comment:
I was not sure - because in the next line you use it print all installed versions from brew...
Not sure if it any benefit...